src/playermat/Playmat.ttslua

```diff
@@ -257,10 +257,10 @@ function doUpkeep(_, color, alt_click)
     end
   end
 
-  -- flip investigator mini-card if found
-  -- flip summoned servitor mini-cards (To-Do: don't flip all of them)
+  -- flip investigator mini-card and summoned servitor mini-card
+  -- (all characters allowed to account for custom IDs - e.g. 'Z0000' for TTS Zoop generated IDs)
```

Comment:
Can we fix the -promo issue here as well, as long as we're changing this?

Reply:
We could account for it here, but I thought we wanted to do that by having them as states of the same card.
(With both having the same ID and also states for the mini card)